### PR TITLE
Create a policy that installs kyverno using its OLM operator

### PR DESCRIPTION
**Pull request by gparvin**
Kyverno has an operator available now on OperatorHub.  This policy can be used to deploy it more easily on OpenShift clusters.

### CODE REVIEW

File: community/README.md [hunk 1]
```diff
@@ -133,7 +133,8 @@ Policy  | Description | Prerequisites
...
 [Policy to setup ODF](./CM-Configuration-Management/policy-odf.yaml) | Use this policy to install and configure the OpenShift Data Foundation | making it work would require e.g. on AWS m5 instances would be required. Requires OpenShift 4.6 or later.
-[Policy to install Kyverno](./CM-Configuration-Management/policy-install-kyverno.yaml) | Use this policy to install Kyverno | Consult the following [link](https://github.com/kyverno) to get more information about Kyverno.
+[Policy to install Kyverno's Helm Chart](./CM-Configuration-Management/policy-install-kyverno.yaml) | Use this policy to install Kyverno | Consult the following [link](https://github.com/kyverno) to get more information about Kyverno.
```
Comment (dhaiducek) [inline community/README.md]:
```suggestion
[Policy to install Kyverno's Helm Chart](./CM-Configuration-Management/policy-install-kyverno.yaml) | Use this policy to install Kyverno | Consult the [Kyverno GitHub](https://github.com/kyverno) to get more information about Kyverno.
```

File: community/README.md [hunk 2]
```diff
@@ -133,7 +133,8 @@ Policy  | Description | Prerequisites
...
+[Policy to install Kyverno operator](./CM-Configuration-Management/policy-kyverno-operator.yaml) | Use this policy to install Kyverno's OLM Operator on OpenShift | Consult the following [link](https://github.com/kyverno) to get more information about Kyverno.
```
Comment (dhaiducek) [inline community/README.md]:
```suggestion
[Policy to install Kyverno operator](./CM-Configuration-Management/policy-kyverno-operator.yaml) | Use this policy to install Kyverno's OLM Operator on OpenShift | Consult the [Kyverno GitHub](https://github.com/kyverno) to get more information about Kyverno.
```

File: community/CM-Configuration-Management/policy-kyverno-operator.yaml [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+  apiGroup: policy.open-cluster-management.io
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+  name: placement-policy-kyverno-operator
+spec:
+  clusterConditions:
+  - status: "True"
+    type: ManagedClusterConditionAvailable
```
Comment (dhaiducek) [inline community/CM-Configuration-Management/policy-kyverno-operator.yaml]: I think the convention now is to leave this out?

**Review by dhaiducek — COMMENTED** [review]
A couple nits about the links and a question about the PlacementRule, but LGTM!

That EnterpriseKyverno manifest is...large. 😄

File: community/README.md (hunk 2 above)
Comment (gparvin) [reply community/README.md]: fixed

File: community/README.md (hunk 1 above)
Comment (gparvin) [reply community/README.md]: fixed

File: community/CM-Configuration-Management/policy-kyverno-operator.yaml (hunk 3 above)
Comment (gparvin) [reply community/CM-Configuration-Management/policy-kyverno-operator.yaml]: yes, agreed.  I took it out

**Review by dhaiducek — APPROVED** [review]
LGTM! Thanks, @gparvin!

**Pull request merged**